include/lsst/afw/fits.h

```diff
@@ -109,7 +109,8 @@ std::string makeLimitedFitsHeader(lsst::daf::base::PropertySet const& metadata,
  *  Throw a FitsError exception if the status of the given Fits object is nonzero.
  */
 #define LSST_FITS_CHECK_STATUS(fitsObj, ...) \
-    if ((fitsObj).status != 0) throw LSST_FITS_EXCEPT(lsst::afw::fits::FitsError, fitsObj, __VA_ARGS__)
+  if ((fitsObj).status != 0) { auto except =  LSST_FITS_EXCEPT(lsst::afw::fits::FitsError, fitsObj, __VA_ARGS__); \
```

Comment:
Is there a reason where there are two spaces after the `=` here and two spaces before the `=` below?

Reply:
the reason is that clang-format for some reason borked on this on my machine. Will fix manually